pkg/controller/bundlec_test/zz_plumbing_for_test.go

```diff
@@ -327,6 +328,47 @@ func (tc *testCase) verifyPlugins(t *testing.T) {
 	assert.Empty(t, unexpectedInvocations, "observed plugin invocations that were not expected")
 }
 
+func (tc *testCase) verifyPluginStatuses(t *testing.T) {
+	var bundle *smith_v1.Bundle
+	for _, action := range tc.smithFake.Actions() {
+		bundleUpdate, ok := action.(kube_testing.UpdateAction)
+		if !ok {
+			continue
+		}
+		bundle = bundleUpdate.GetObject().(*smith_v1.Bundle)
+		// Keep iterating to get the latest action (should be only one actually)
+	}
+	if bundle == nil {
+		t.Log("No Bundle updates found, not checking plugin statuses")
+		return
+	}
+	pluginNames := map[smith_v1.PluginName]struct{}{}
+	for _, res := range tc.bundle.Spec.Resources {
+		if res.Spec.Plugin == nil {
+			continue
+		}
+		pluginNames[res.Spec.Plugin.Name] = struct{}{}
+	}
+	assert.Len(t, bundle.Status.PluginStatuses, len(pluginNames))
+nextPlugin:
```

Comment:
not a fan of goto operator, but for tests fine.

Reply:
I think this is better than an inline anon function and a `return`.